claim/gql_mutations.py

```diff
@@ -369,6 +369,7 @@ def async_mutate(cls, user, **data):
                 return [{
                     'message': _("claim.mutation.duplicated_claim_code") % {'code': data['code']},
                 }]
+            is_claim_code_autogenerated = data.get("autogenerate_code", None)
```

Comment:
The default value for boolean field should be `False`. `None` works too, but it does't indicate the actual type of the field